### ISSUE TITLE
[QUESTION] Inaccessible optionLabel with nested query field on useSelect

### ISSUE THREAD

**Issue by emehmet** [pankod/refine#2376]
Hi,

I have a code like this;

```  
const { selectProps, queryResult } = useSelect({
    resource: 'getUserAssignedPoints',
    optionLabel: 'TITLE',
    optionValue: '_id',
    metaData: {
      fields: ['_id', { LOC: [{ properties: ['TITLE'] }] }],
    },
  });
```

I can't reach TITLE in nested field from optionLabel. How can I do it?

**Comment by omeraplak** [comment]
Hey @emehmet 👋,
Thank you for contacting us.

We need to add NamePath support but for now you can try this ⚡️

```tsx
const {
    selectProps,
    queryResult,
    defaultValueQueryResult
  } = useSelect();

  const allOptions = [
    ...(queryResult.data?.data || []),
    ...(defaultValueQueryResult.data?.data || [])
  ];

```

```tsx
 <Select
  {...categorySelectProps}
  options={allOptions.map((p) => ({
     label: `${p.nested.title}`,
     value: p.id
   }))}
/>
```

**Comment by emehmet** [comment]
Hey @omeraplak ,

Thank you for your solution!

**Comment by omeraplak** [comment]
Hey @emehmet ,
We've released `@pankod/refine-core@3.64.0`. Could you approve this feature?

```tsx
    useSelect({
        resource: "posts",
        optionLabel: "nested.title",
        optionLabel: "nested.id",
    });
```

**Comment by emehmet** [comment]
Hi @omeraplak,

It works perfectly. Thank you very much!
